### ISSUE TITLE
Update Devcontainer to include RUST

### ISSUE THREAD

**Issue by SebastianSchildt** [eclipse/kuksa.val#355]
It might be a good idea to update the devcontainer to also install a suitable rust version and request RUST extension in Visualstudio code 
(also check if it already includes pipenv)

https://github.com/eclipse/kuksa.val/tree/master/.devcontainer

**Comment by AlexanderLanin** [comment]
How about adding it as a feature? It's rather trivial. Extend devcontainer.json:
```
"features": {
    "ghcr.io/devcontainers/features/rust:1": {}
}
```
License: MIT

Here is some background information: https://containers.dev/


Downside: using that without VS Code might require one extra step before starting the devcontainer (calling `devcontainer-cli`). However that's probably not trivial anyway, so one extra step doesn't hurt. People not using VS Code are potentially not using the devcontainer either.

**Comment by SebastianSchildt** [comment]
I am not really much familiar with the "special" features of vscidedev containers, however at this point in time we would probably take "any" improvement, because I think currently you get  container, that basically lacks dependencies to build anything (I think it is still from when we only had the c++ val-server, and that one grabs most of it's dependencies via cmake anyway).

A dev container IS however (even in its current form) useful especially for WIndows and Mac users, because well you don't need to think about, how you get anything in a (supported) Linux environment, VSCode will just ask "shall I reopen this in devocntainer" and of you go.

tldr; You seem to know more about devcontainers than some of us. Will take any improvement 😁

**Comment by SebastianSchildt** [comment]
Dev Container removed in #572
